src/nodes.coffee

```diff
@@ -3779,15 +3807,34 @@ exports.Code = class Code extends Base
     else
       name
 
-  astProperties: (o) ->
+  methodAstProperties: (o) ->
     return
+      static: no
```

Comment:
In TDD style, hardcode `static` and `computed` for now and will make them dynamically calculated once we get there/have tests that require it